json-core/src/main/java/io/micronaut/json/JsonMapper.java

```diff
@@ -201,6 +202,29 @@ default Processor<byte[], JsonNode> createReactiveParser(@NonNull Consumer<Proce
      */
     <T> byte[] writeValueAsBytes(@NonNull Argument<T> type, @Nullable T object) throws IOException;
 
+    /**
+     * Write an object as String.
+     *
+     * @param object The object to serialize.
+     * @return The serialized encoded json.
+     * @throws IOException IOException
+     */
+    default String writeValueAsString(@Nullable Object object) throws IOException {
+        return writeValueAsString(object, StandardCharsets.UTF_8);
+    }
+
+    /**
+     * Write an object as String.
+     *
+     * @param object The object to serialize.
+     * @param  charset The {@linkplain java.nio.charset.Charset charset} to be used to decode the {@code bytes}.
+     * @return The serialized encoded json.
+     * @throws IOException IOException
+     */
+    default String writeValueAsString(@Nullable Object object, @NonNull Charset charset) throws IOException {
```

Comment:
remove this, jackson doesnt have it, nobody will use it, and it doesnt make sense for json

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/9402/commits/8f59667c2da700ad9bcfd4bc4f2d80924e76ab6c.